### PR TITLE
Bitcoin loading svg icon

### PR DESCRIPTION
**Pull request by Draichi**
## Screenshot

Recording 2022-02-01 at 02 42 05

This is a suggestion on how to show something cool to the user while he waits

Fix: #680 

## What needs to be done

The logic in this component is a copy of https://github.com/mempool/mempool/pull/713. Although there is an error on the `map` function when used in `start.component.ts`:
image

```
Cannot find name 'map'. Did you mean 'Map'?ts(2552)
lib.es2015.collection.d.ts(36, 13): 'Map' is declared here.
```

Couldn't find a solution yet

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -83,6 +83,7 @@
     "domino": "^2.1.6",
     "echarts": "^5.1.2",
     "express": "^4.17.1",
+    "gsap": "^3.9.1",
```
Comment (wiz) [inline frontend/package.json:86]: Concept NACK for adding new trusted third party npm dependencies

**Comment by wiz** [comment]
Thanks for the PR, but to be honest, I don't think this animation matches the design style of the project...

**Comment by softsimon** [comment]
This should be the dashboard loading icon:

https://jsfiddle.net/miguelmedeiros/7nt8dy6b/2/

**Comment by wiz** [comment]
Another contributor of ours made an animated version of the mempool logo entirely in CSS that we had considered using for this purpose, maybe you can look at integrating that instead? And you don't need any new npm dependencies

**Comment by softsimon** [comment]
It seems like the `specialEvent` feature is deleted in this PR and the `mempoolLoadingStatus$` is not working.. So this is not really a working PR ready for review.

If you instead use the animated mempool logo and solve the loading status detection, it would be something we would consider merging.

**Pull request closed without merging**